.changeset/shaggy-mugs-speak.md

```diff
@@ -0,0 +1,7 @@
+---
```

Comment:
Also add some words about that the `title` of the `DocumentSection` now is `nullable`

Reply:
I guess this is not a `patch`, it's a breaking change. Returning Nullable `title` most likely break themes.

Reply:
Yeah its also only added for `dev` => alpha version, but yes you are right we should mark it as `major`

(though not sure if changeset it will handle correctly and not release v5-alpha, but we will see and could adjust the changeset pr)

Reply:
I've added a separate changeset with the major type change to separate the updates